analyzers/its/create-issue-reports.ps1

```diff
@@ -13,16 +13,37 @@ $thrd.CurrentUICulture = $thrd.CurrentCulture
 [void][System.Reflection.Assembly]::LoadWithPartialName("System.Web.Extensions")
 $jsonserial= New-Object -TypeName System.Web.Script.Serialization.JavaScriptSerializer
 $jsonserial.MaxJsonLength = 100000000
+$tempFileNameRegex = '^.*\.(NETFramework|NETCoreApp),Version='
```

Comment:
IMO `tempFileNameRegex` can be inside `Create-FullUriForIssue` because it's very specific to the URI